testsuite/manualmode/src/test/java/org/wildfly/core/test/standalone/mgmt/ManagementInterfaceResourcesCommunityTestCase.java

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright The WildFly Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.wildfly.core.test.standalone.mgmt;
+
+import static org.jboss.as.controller.client.helpers.Operations.createAddress;
+import static org.jboss.as.controller.client.helpers.Operations.createWriteAttributeOperation;
+import static org.jboss.as.test.integration.management.util.ServerReload.executeReloadAndWaitForCompletion;
+
+import org.jboss.as.test.integration.management.util.ServerReload;
+import org.jboss.as.version.Stability;
+import org.jboss.dmr.ModelNode;
+import org.wildfly.common.function.ExceptionRunnable;
+import org.wildfly.core.testrunner.ManagementClient;
+import org.wildfly.core.testrunner.ServerSetupTask;
+import org.wildfly.test.stability.StabilityServerSetupSnapshotRestoreTasks;
+
+/**
+ * Test case to test resource limits and clean up of management interface connections.
+ *
+ * This test case uses attributes defined directly on the HTTP management interface resource for configuration.
+ *
+ * @author <a href="mailto:darran.lofthouse@jboss.com">Darran Lofthouse</a>
+ */
+public class ManagementInterfaceResourcesCommunityTestCase extends AbstractManagementInterfaceResourcesTestCase {
+
+    /*
+    * Attribute names
+    */
+    private static final String BACKLOG_ATTRIBUTE = "backlog";
+    private static final String CONNECTION_HIGH_WATER_ATTRIBUTE = "connection-high-water";
+    private static final String CONNECTION_LOW_WATER_ATTRIBUTE = "connection-low-water";
+    private static final String NO_REQUEST_TIMEOUT_ATTRIBUTE = "no-request-timeout";
+
+    private static final ModelNode HTTP_INTERFACE_ADDRESS = createAddress("core-service", "management", "management-interface", "http-interface");
+
+    protected void runTest(int noRequestTimeout, ExceptionRunnable<Exception> test) throws Exception {
+        controller.start();
+        ManagementClient client = controller.getClient();
+
+        ServerSetupTask task = new ManagementInterfaceSetUpTask(noRequestTimeout);
+
+        try {
+            task.setup(client);
+            test.run();
+            controller.reload();
```

Comment:
This reload here seems unnecessary, doesn't it?

Reply:
I am not sure at the moment, the original tests also had this reload.

The problem with this test is we deliberately exhaust the http management interface connection to test that this configuration is working and then we try and use the connection to perform further updates.